specification/graphrbac/data-plane/stable/1.6/graphrbac.json

```diff
@@ -2985,6 +3054,52 @@
         }
       },
       "description": "Server response for Get tenant domains API call."
+    },
+    "OAuth2" : {
```

Comment:
As per the linter, having this share a name with the Operation Group will create a naming conflict in the generated SDKs. Left unchanged, AutoRest will automatically rename this "OAuth2Model". Given context, I'm not sure )Auth2 is a very expressive name for this anyway. Maybe something like "GrantRequest" or "GrantPermission"?

Reply:
@marstr , I will change this to Permissions in anticipation that we can add a revoke in the future which does not seem to exist at this stage